src/param.js

```diff
@@ -214,9 +214,14 @@ class List extends Param {
 }
 module.exports.List = (...args) => new List(...args);
 
+const AUTO_PRUNE = Symbol('AUTO_PRUNE');
+const isAutoPrune = input => input[AUTO_PRUNE] === true;
+module.exports.isAutoPrune = isAutoPrune;
```

Comment:
Can we move this onto the parameter itself? And then check with instanceof? Or was instanceof problematic for some reason?

Reply:
This is not where I meant the `Symbol` btw. I meant on the `result` (have have now changed my mind as described above). I'd just set it as `this.autoPrune = ...` and then use instanceof is that is possible.